src/main/java/Dude.java

```diff
@@ -0,0 +1,213 @@
+import java.io.*;
```

Comment:
Imported classes should be listed explicitly 😁 
```suggestion
import java.io.BufferedReader;
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;
```